### DISCUSSION TITLE
How can I add a custom field to the Post content type?

### DISCUSSION THREAD

**Discussion by nathanaskinner** [flyntwp/flynt#544] [Q&A]
I'd like to add a custom field to the Post Type for to use as a  'Read More' override. What would be the best way to go about this?

**Comment by aaronmeder** [comment]
Hey @nathanaskinner 

You could just add the custom field by adding a Field Group through ACF's admin ui:
![Screenshot 2024-03-27 at 18 39 09](https://github.com/flyntwp/flynt/assets/5564163/21518303-2da9-422f-b1a7-e6d35d5958a9)

### Bonus Tip #1: Change "Read More" Label in Post Listings
If you'd only like to change the default "read more" text in your post grids (latest posts or post archive pages) then that is already prepared for you. Go to "Translatable Options" and change the label inside "Grid Post Archive" or "Grid Post Latest":
![Screenshot 2024-03-27 at 18 43 10](https://github.com/flyntwp/flynt/assets/5564163/2d03d756-c082-4ff5-ba45-751f252c755d)


### Bonus Tip #2: storing the field group in your theme
If you like then you can use ACF's [Local JSON](https://www.advancedcustomfields.com/resources/local-json/) functionality to store the field groups you create using the admin ui inside your theme folder as JSON files. This makes it easy to version control them and deploy them to your hostings together with your theme.
The only thing you need to do is to create a "acf-json" folder inside your Flynt directory and save the field group again...you'll see a json containing all the field group information being saved.

Enjoy the evening!
Aaron
